### PR TITLE
sprint-16-release

### PR DESCRIPTION
**Pull request by sethalt**
Stories completed: 993, 1002, 690, 1001, 957, 991, 990, 1003

### CODE REVIEW

File: frontend/e2e/application-temp-outfitters.e2e-spec.ts [hunk 1]
```diff
@@ -72,10 +69,10 @@ describe('Apply for a temp outfitters permit', () => {
...
   });
 
-  fieldValidation.validateFileUploadField('section-guide-identification', 'xls');
-  fieldValidation.validateFileUploadField('section-acknowledgement-of-risk');
-  fieldValidation.validateFileUploadField('section-liability-insurance');
-  fieldValidation.validateFileUploadField('section-operating-plan');
+  fieldValidation.validateFileUploadField('guide-document', 'xls');
+  fieldValidation.validateFileUploadField('acknowledgement-of-risk-form', 'pdf');
+  fieldValidation.validateFileUploadField('insurance-certificate', 'pdf');
+  fieldValidation.validateFileUploadField('operating-plan', 'pdf');
```
Comment (lauraGgit) [inline frontend/e2e/application-temp-outfitters.e2e-spec.ts]: @colinpmacarthur could you confirm that these are correct?

File: frontend/src/app/application-forms/fields/fax.component.ts [hunk 2]
```diff
@@ -11,15 +12,15 @@ export class FaxComponent implements OnInit {
   formName: string;
   fax: any;
 
-  constructor(private formBuilder: FormBuilder) {}
+  constructor(private formBuilder: FormBuilder, public afs: ApplicationFieldsService) {}
```
Comment (lauraGgit) [inline frontend/src/app/application-forms/fields/fax.component.ts]: we may want to keep this as ApplicationFieldService spelled out.

**Comment by lauraGgit** [comment]
@dubharmonic could you provide screenshots in the summary of the 404s as well as the headers being set online?

**Comment by dubharmonic** [comment]
Added screenshots. The CORS headers are being set for the API in the middleware here: https://github.com/18F/fs-intake-module/blob/master/server/src/middleware.es6

**Comment by lauraGgit** [comment]
@dubharmonic do you mind just adding that in the summary? Then we will be good to go? "The CORS headers are being set for the API in the middleware here: https://github.com/18F/fs-intake-module/blob/master/server/src/middleware.es6"

**Pull request merged**